dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/tests/Audacia.CodeAnalysis.Analyzers.Test/Rules/ControllerActionProducesResponseTypeAnalyzerTests.cs

```diff
@@ -129,6 +129,55 @@ public string Get()
             VerifyDiagnostic(testCode, expectedDiagnostic);
         }
 
+        [TestMethod]
+        public void Diagnostics_For_Controller_With_No_ProducesResponseType_And_IActionResult_ReturnType_Attribute()
+        {
+            const string testCode = @"
+                using System.Threading.Tasks;
+                using Microsoft.AspNetCore.Mvc;
+
+                namespace ConsoleApplication1
+                {
+                    public class TestController : ControllerBase
+                    {
+                        [HttpGet]
+                        public IActionResult Get()
+                        {
+                            return 'hello';
```

Comment:
question: does this code actually compile? I can't imagine it would be able to implicitly convert `'hello'` to an `IActionResult`?

Reply:
No this won't be compiled during the test but I get your point. I will change the code.